src/SubscriptionBuilder.php

```diff
@@ -254,6 +259,7 @@ protected function buildPayload()
         return array_filter([
             'billing_cycle_anchor' => $this->billingCycleAnchor,
             'coupon' => $this->coupon,
+            'expand' => ['latest_invoice.payment_intent'],
```

Comment:
What does this do?

Reply:
It makes use of the `expand` functionality from the Stripe API to retrieve data from objects in one request. By default when creating a new subscription, the `latest_invoice` parameter contains a string with its ID. By using the `expand` here we retrieve the actual object as well as the payment intent object so we can immediately use it below when validating the payment.